docs/pull_hf_models.md

```diff
@@ -1,6 +1,28 @@
 # OVMS Pull mode {#ovms_docs_pull}
 
-This documents describes how leverage OpenVINO Model Server (OVMS) pull feature to automate deployment configuration with Generative AI models from OpenVINO organization in HuggingFace (HF). This approach assumes that you are pulling from [OpenVINO organization](https://huggingface.co/OpenVINO) from HF. If the model is not from that organization, follow steps described in [this document](../demos/common/export_models/README.md).
+This documents describes how leverage OpenVINO Model Server (OVMS) pull feature to automate deployment configuration with Generative AI models from or outside of OpenVINO organization in HuggingFace (HF). When pulling from [OpenVINO organization](https://huggingface.co/OpenVINO) from HF no additional steps are required. However, when pulling models outside of the OpenVINO organization you have to install additional python dependencies when using baremetal execution so that optimum-cli is available for ovms executable or build the OVMS python container for docker deployments.
+
+Specific OVMS pull mode example for models outside of OpenVINO organization is described in section `2. Download the preconfigured models using ovms --pull option for models outside OpenVINO organization` in the [RAG demo](https://github.com/openvinotoolkit/model_server/blob/main/demos/continuous_batching/rag/README.md)
+
+## Pulling models outside of OpenVINO organization
```

Comment:
This should be separated from GGUF model pulling.
The options should be :
- pulling preconfigured models in IR format from OpenVINO organization
- pulling GGUF models
- pulling models with automatic conversion and quantization (requires optimum-cli). Include additional consideration like longer time for deployment and pulling model data (original model) from HF

We should include instructions how to prepare the image with optimum-cli and how to prepare the package with optimum-cli. It should be a separate page because it will be common for pulling instructions and starting instructions.
@atobiszei